.github/workflows/amplify_integration_tests.yaml

```diff
@@ -158,49 +166,51 @@ jobs:
         timeout-minutes: 30
         run: |
           chromedriver --port=4444 &
-          melos exec -c 1 --scope ${{ matrix.scope }} -- \$MELOS_ROOT_PATH/build-support/integ_test.sh -d web-server
+          aft exec --include=${{ matrix.scope }} -- "<AFT_ROOT>/build-support/integ_test.sh" -d web-server
 
-  macos:
-    runs-on: macos-latest
-    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
-    permissions:
-      id-token: write
-      contents: read
-    strategy:
-      fail-fast: false
-      matrix:
-        scope:
-          - "amplify_analytics_pinpoint_example"
-          - "amplify_api_example"
-          # TODO(dnys1): macOS tests are failing for some reason
-          # - "amplify_authenticator_example"
-          - "amplify_auth_cognito_example"
-          - "amplify_storage_s3_example"
-          - "amplify_db_common_example"
-    steps:
-      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # 3.1.0
-        with:
-          persist-credentials: false
-          submodules: true
+  # TODO(dnys1): Re-enable with self-hosted runners
```

Comment:
Why do these have to be disabled?

Reply:
macOS integ tests were running on Flutter 3.3 still (Dart 2.18) due to the fact that Flutter 3.7 dropped support for the OpenGL rendering backend that GitHub Actions runners needed. As of now, it's infeasible to support them any longer. This was called out when Flutter 3.7 came out that we'd need to adapt to self-hosted runners of some sort.